File: 2784. Check if Array is Good/Solution.py
class Solution:
    def isGood(self, nums: List[int]) -> bool:
        high = max(nums)
        if len(nums) != high + 1: return False
        counter = [0 for i in range(high + 1)]
        for n in nums: counter[n] += 1
        for i in range(1, high):
            if counter[i] != 1: return False
        return counter[high] == 2

File: 2784. Check if Array is Good/Solution.java
class Solution {
    public boolean isGood(int[] nums) {
        int l = nums.length, high = 0;
        for (int n : nums) high = Math.max(high, n);
        if (l != high + 1) return false;
        int[] counter = new int[high + 1];
        for (int n : nums) counter[n]++;
        for (int i = 1; i < high; i++) {
            if (counter[i] != 1) return false;
        }
        return counter[high] == 2;
    }
}
